src/main/java/org/apache/mesos/scheduler/plan/DefaultPhase.java

```diff
@@ -182,6 +181,10 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return ReflectionToStringBuilder.toString(this);
+        return "DefaultPhase{" +
```

Comment:
Any reason for making this manual? It looks like `byId` is no longer being printed. If that's intentional then maybe mention that in a comment.